pandatool/src/miscprogs/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_executable(bin2c binToC.cxx binToC.h)
+target_link_libraries(bin2c p3progbase)
+install(TARGETS bin2c EXPORT Tools COMPONENT Tools DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Comment:
Let's not build (or install) bin2c; we stopped doing this in makepanda because it conflicted with bin2c provided by other packages.  It doesn't seem like a Panda-specific tool anyway.

Reply:
Should we be deleting binToC.cxx? I'd rather build the file (but perhaps not install it) if it exists. If we're straight-up not building it, we should delete it so we don't even bother maintaining it.

Reply:
I'd like this being switchable (the default being off), like pview. This tool is useful for compiling in bams (e. g. default font), while the source files can be kept in an editable format and be converted to the then current bam version.

Reply:
We could build it but not install it.  Then people who compile Panda from source (the kind who are likely to compile-in default fonts) can still use it from the build directory.

However, are there really no third-party tools that can do the same thing?  It seems like a very generic thing.  I'd even be open to just ripping this out into its own repository and putting it up on PyPI so those who need it can `pip install bin2c` or something.

Reply:
Is the question whether something like this is needed at build time? Because the `ConcatenateToCXX.cmake` script that I whipped up for adding the Interrogate preamble does all this and more. If we want to keep default fonts in non-C format (ttf and bam) the concatenate scrpit would work for those too and we can remove bin2c.

Reply:
Oo, I like the idea of being able to specify a font file to CMake and having it automatically implement the bin2c step using a CMake script.